src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Usage/CSharpPreferGenericOverloads.Fixer.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Analyzer.Utilities.Extensions;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Operations;
+using Microsoft.CodeAnalysis.Simplification;
+using Microsoft.NetCore.Analyzers.Usage;
+using static Microsoft.NetCore.Analyzers.Usage.PreferGenericOverloadsAnalyzer;
+
+namespace Microsoft.NetCore.CSharp.Analyzers.Usage
+{
+    /// <summary>
+    /// CA2263: <inheritdoc cref="NetCore.Analyzers.MicrosoftNetCoreAnalyzersResources.PreferGenericOverloadsTitle"/>
+    /// </summary>
+    [ExportCodeFixProvider(LanguageNames.CSharp), Shared]
+    public sealed class CSharpPreferGenericOverloadsFixer : PreferGenericOverloadsFixer
+    {
+        protected override async Task<Document> ReplaceWithGenericCallAsync(Document document, IInvocationOperation invocation, CancellationToken cancellationToken)
+        {
+            if (!RuntimeTypeInvocationContext.TryGetContext(invocation, out var invocationContext))
+            {
+                return document;
+            }
```

Comment:
Is this condition expected to be hit? I assume it shouldn't because the analyzer should have already filtered valid invocations only?

Reply:
I would not expect it to be hit, but I was not sure if there is a corner case, where the symbols are found in the analyzer, but are not found when the fixer is run. Also, this silences the nullability warnings when I use the context later.

Should I remove the check or add a comment that this branch is very unlikely to be hit?

Reply:
The code is probaby okay as is. I was just making sure this is not expected to be hit.